src/polymer-project.ts

```diff
@@ -214,26 +224,6 @@ export class PolymerProject {
     return new HtmlRejoiner(this);
   }
 
-  /**
-   * Returns the a `Transform` that runs Hydrolysis analysis on the files, for
-   * use by the bundler transform. This transform must only be used in one
-   * stream.
-   */
-  get analyze(): Transform {
```

Comment:
It seemed like these were built when the analyzer/bundler were only used as streams. Now that the analyzer is a part of the public interface (service-worker generation calls `analyzer.analyzeDependencies()`) the getter/alias becomes unnecessary and the `-er` form became more fitting.